phpcs.xml

```diff
@@ -2,6 +2,8 @@
 <ruleset name="WordPress Coding Standards">
 	<description>WooCommerce dev PHP_CodeSniffer ruleset.</description>
 
+	<file>.</file>
```

Comment:
What this line does?

Reply:
In `--help` describes as "One or more files and/or directories to check".
But not sure if we can always check for all files, since we also use it on pre-commit hooks, have you tested and checked if pre-commit still works as expected?

Reply:
If using PHPCS 3.5.0, you may want to add `filter=gitstaged` to the precommit hook, so it will only check the files staged ;-)

Reply:
> What this line does?

This is needed to make the change to show only the relevant bit of file paths in reports work. I don't this will have any other impact. In the pre-commit hook we pass a list of files for PHPCS to check so it won't check all the files. I will look into @jrfnl suggestion to use `filter=gitstaged`.

Reply:
> This is needed to make the change to show only the relevant bit of file paths

This is not the `basepath`?

Reply:
Since we use some libraries on NPM to create pre-commit hooks we don't need `filter=gitstaged`. But seems Rodrigo said that still works fine with our pre-commit, so all looks fine.

Reply:
> This is needed to make the change to show only the relevant bit of file paths in reports work. I don't this will have any other impact.

FYI:

This is irrelevant for the _"show only the relevant bit of file paths in reports work"_.

As @claudiosanches found in the docs: it tells PHPCS which files to scan and with the current setting this is "all files".

The `<exclude-pattern>`s will limit that again, as well as the `extensions` setting.

Any command-line path restrictions will limit it further.